src/rprblender/ui/material.py

```diff
@@ -122,6 +122,9 @@ class RPR_MATERIAL_PT_surface(RPR_MaterialOutputSocket):
     def draw(self, context):
         layout = self.layout
 
+        row = layout.row()
+        row.prop(context.material, 'pass_index')
```

Comment:
I'm not sure it's needed here.
This parameter is already present in Material tab -> Viewport Display section ->Settings subsection, enabled in RPR UI.

Reply:
Hmmm ok, I didn't see it in there, kinda hidden.  But I'll remove.